crypto/ex_data.c

```diff
@@ -367,7 +368,8 @@ void CRYPTO_free_ex_data(int class_index, void *obj, CRYPTO_EX_DATA *ad)
         if (storage != NULL)
             f = storage[i];
         else {
-            CRYPTO_THREAD_write_lock(global->ex_data_lock);
+            if (!CRYPTO_THREAD_write_lock(global->ex_data_lock))
+                continue;
```

Comment:
I'm unsure what the best way to handle this error is. We could also just ignore it. Maybe add some comment?

Reply:
Just leave it as it is.